aerogear/keycloak-metrics-spi
# Feature/maven wrapper

## Motivation
To ensure platform independence for Maven builds and to align with best practices for encoding standards.

## What
Added Maven wrapper and set UTF-8 as default encoding.

## Why
To make the build process more consistent across different platforms and to avoid encoding-related issues.

## How
- Added Maven wrapper files (mvnw, mvnw.cmd, .mvn/wrapper/)
- Configured Maven to use UTF-8 encoding by default (<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding> in pom.xml)

# Verification Steps
1. Clone the repository.
2. Navigate to the project directory.
3. Verify that the Maven wrapper files (mvnw, mvnw.cmd) are present.
4. Open pom.xml and confirm <project.build.sourceEncoding> is set to UTF-8.
5. Run a build using the Maven wrapper (./mvnw clean install) and ensure the build completes successfully on different platforms (Windows, macOS, Linux) and no warning appeared for source encodig.

## Checklist:

## Progress

## Additional Notes

No additional notes.
